fr.yml

```diff
@@ -62,367 +62,365 @@ tray:
-        titleInclusion: "Liste d'inclusion de processus personnalisés"
-        descriptionInclusion: "Une liste personnalisée de noms/chemins de processus supplémentaires que le mod va inclure dans le ciblage. Cette liste est ajoutée à la liste d'inclusion que l'auteur du mod a défini pour déterminer quel processus cibler. Pour chaque processus, le mod est chargé si le chemin du processus correspond à l'un de ceux inclus et ne correspond à aucune de ceux exclus."
-        titleExclusion: "Liste de processus exclus personnalisée"
-        descriptionExclusion: "Une liste personnalisée de noms/chemins de processus supplémentaires que le mod va exclure du ciblage. Cete liste est ajoutée à la liste d'exclusion que l'auteur du mod a défini pour déterminer quel processus cibler. Pour chaque processus, le mod est chargé si le chemin du processus correspond à l'un de ceux inclus et ne correspond à aucune de ceux exclus."
-        processListPlaceholder: "Noms/chemins des processus, un par ligne, par exemple :"
-        invalidCharactersWarning: ~
-        saveButton: "Sauvegarder"
-      includeExcludeCustomOnly:
-        title: "Ignorer les listes d'inclusion/exclusion de mods"
-        description: "Ignorer les listes d'inclusion/exclusion de processus de mods et n'utiliser que les listes personnalisées ci-dessus."
-      patternsMatchCriticalSystemProcesses:
-        title: ~
-        description: ~
+        titleInclusion: "Liste d'inclusion de processus personnalisée"
+        descriptionInclusion:  "Une liste personnalisée de noms/chemins de fichiers exécutables supplémentaires que le mod ciblera.  La liste est ajoutée à la liste d'inclusion définie par l'auteur du mod pour déterminer quels processus sont ciblés."
+        titleExclusion: "Liste d'exclusion de processus personnalisée"
+        descriptionExclusion: "Une liste personnalisée de noms/chemins de fichiers exécutables supplémentaires que le mod exclura du ciblage. La liste est ajoutée à la liste d'exclusion définie par l'auteur du mod pour déterminer quels processus sont exclus."
+        processListPlaceholder: "Noms/chemins de processus, un par ligne, par exemple :"
+        invalidCharactersWarning: "La liste de processus contient des caractères invalides qui seront supprimés :  {{invalidCharacters}}"
+        saveButton: "Enregistrer"
+      includeExcludeCustomOnly: 
+        title: "Ignorer les listes d'inclusion/exclusion du mod"
+        description: "Ignorer les listes d'inclusion/exclusion de processus du mod et utiliser uniquement les listes personnalisées ci-dessus."
+      patternsMatchCriticalSystemProcesses: 
+        title: "Prendre en compte les motifs de la liste d'inclusion pour les processus système critiques"
+        description: "Par défaut, Windhawk ne charge les mods dans les processus système critiques que si le processus est inclus sans motifs, par ex. <0>critical. exe</0>, pas <0>*</0> ou <0>*. exe</0>.  Pour plus de détails, consultez la documentation."
```

Comment:
Remove the spaces in `. exe`.

Reply:
Also, missing the documentation link tag.
- English: "...please refer to <1>the documentation</1>."
- French: "...consultez la documentation." (missing `<1>` tags - the link will not work)